### ISSUE TITLE
[Bug] High-lighting is broken when used in a markdown link

### ISSUE THREAD

**Issue by aledeg** [shaarli/Shaarli#1735]
This is a follow-up issue of #1729 but I do not know if it is related.

Here is the content of my markdown:
```md
[Music | Los Bitchos](https://losbitchos.bandcamp.com/music)
```

It renders perfectly
![Screenshot_2021-04-07 Mes liens - perso](https://user-images.githubusercontent.com/3056148/113923301-68060880-97b6-11eb-8249-55f2c0e76dd5.png)

until I search for a word which is in the link (Here, I searched for *losbitchos*).
![Screenshot_2021-04-07 Search losbitchos - Mes liens - perso](https://user-images.githubusercontent.com/3056148/113923408-81a75000-97b6-11eb-8741-f7d81cac6dd8.png)

As the link is broken, I cannot click on it.

----
I am currently using Shaarli 0.12.1 with PHP 7.4.

**Comment by ArthurHoaro** [comment]
I think this one was fixed by #1696. In any case it seems to be working on the master branch.

![](https://i.imgur.com/NXuYzIf.png)

**Comment by nodiscc** [comment]
yep, duplicate of https://github.com/shaarli/Shaarli/issues/1681
